README.md

```diff
@@ -389,6 +390,23 @@ Running the build
     and you are behind a non-transparent proxy server,
     [configure your proxy server in Maven](http://maven.apache.org/settings.html#Proxies).
 
+ Note: if you get the following error: 
+ 
+    ```shell
+    [ERROR] Failed to execute goal org.codehaus.mojo:jaxb2-maven-plugin:1.5:schemagen (generate-xsd) on project kie-remote-jaxb-gen: Failed to generate schema: InvocationTargetException: Class not foundcom/sun/tools/apt/Main.class -> [Help 1]
+    ``` 
```

Comment:
The 6.2.x branch is no longer supported. There will be no more releases from that branch, so even if we cherry-picked the change, there would be no new release with that fix. If you are building from sources (which you do it seems), you can easily cherry-pick that commit into your own branch.